README.md

```diff
@@ -18,7 +18,7 @@ This repository lists resources and projects, highlighting official and communit
 - [About MDN](https://developer.mozilla.org/en-US/about) - MDN's mission, team, partners, and projects.
 - [Curriculum](https://mdn.dev/curriculum) - Structured learning resources and modules for web development education.
 - [Content Repo](https://github.com/mdn/content) - Source repository for MDN's (en-US) documentation and examples.
-- [Platform Repo](https://github.com/mdn/yari) - Repository for the MDN platform, tools and site infrastructure.
+- [Yari](https://github.com/mdn/yari) - MDN's React-based frontend, including tools for building the site.
```

Comment:
```suggestion
- [Yari](https://github.com/mdn/yari) - MDN's React-based frontend, including tools for managing content.
```

Reply:
Shouldn't it be rari now? How much longer to completely switch to rari repo?